src/content/docs/tutorial-create-alerts/improve-with-alerts.mdx

```diff
@@ -0,0 +1,94 @@
+---
+title: "Manage your alert quality"
+metaDescription: "How manage your alerts and notifications"
+---
+
+import AQMdash from 'images/alerts_screenshot-crop_AQM-dash.webp'
+
```

Comment:
This doc feels like the real meaty heart of the tutorial! I'm worried readers won't make this far though.

Reply:
Agreed, we can iterate on the doc to make it easier to get here, but I don't know that we want to delay the doc another week to do so.